app/lib/common/providers/notifiers/attachments_notifiers.dart

```diff
@@ -36,3 +42,77 @@ class AttachmentsManagerNotifier
     return arg;
   }
 }
+
+class AttachmentDraftsNotifier extends StateNotifier<List<AttachmentDraft>> {
+  final AttachmentsManager manager;
+  final Ref ref;
+
+  AttachmentDraftsNotifier({required this.manager, required this.ref})
+      : super([]);
+
+  /// converts user selected media to attachment draft and adds to state list.
+  /// only supports image/video/audio/file.
+  Future<void> addDraft(File file) async {
+    EasyLoading.show(status: 'Adding draft');
```

Comment:
this is an odd message to show to the user, as they never said anything about doing a "draft". and now that this is a direct interaction API this probably shouldn't really care about adding draft and sending them as two steps ...?!?

Reply:
Will remove this loading indication!

Reply:
Fixed by c6ea65a.